patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -1,16 +1,24 @@
 diff --git a/chrome/android/java/AndroidManifest.xml b/chrome/android/java/AndroidManifest.xml
-index 5a71180c427c05a62e09b3a56c23c50fe03a0b36..ed14dc6aa36e44fbc9c7dd3ec4bfd2ba5da2f99f 100644
+index 5a71180c427c05a62e09b3a56c23c50fe03a0b36..5b9e09209acaeb14bdae918efbc2759c9f1027a7 100644
 --- a/chrome/android/java/AndroidManifest.xml
 +++ b/chrome/android/java/AndroidManifest.xml
-@@ -961,7 +961,7 @@ by a child template that "extends" this file.
+@@ -961,14 +961,14 @@ by a child template that "extends" this file.
          </provider>
  
          <!-- Sync adapter for browser invalidation. -->
 -        <service android:name="org.chromium.chrome.browser.invalidation.ChromeBrowserSyncAdapterService"
-+        <service android:name="org.chromium.chrome.browser.invalidation.BraveBrowserSyncAdapterService"
++<!--        <service android:name="org.chromium.chrome.browser.invalidation.ChromeBrowserSyncAdapterService"
```

Comment:
please put html comments above/below existing lines so there are only additions in the diff and no changes

Reply:
thanks